### DISCUSSION TITLE
How to set xAxis. setTickUnit(2);

### DISCUSSION THREAD

**Discussion by Shuao058** [GSI-CS-CO/chart-fx#635] [Q&A]
![111111111111](https://github.com/fair-acc/chart-fx/assets/98296380/b715e135-85d8-497d-9943-895f5552f97d)
I set the scale interval to 2, but it doesn't seem to work. It's actually 2.5, and the maximum scale value on the x-axis also disappears

**Comment by ennerf** [comment]
The scale is automatically determined by the axis. You would have to overwrite the axis to do a custom scale.

**Reply by Shuao058** [reply]
What is the reason for the disappearance of the maximum number axis value on the x-axis

**Reply by ennerf** [reply]
See #648 regarding the tickUnit. The maximum number disappears if there is not enough margin to fully display it.

**Reply by wirew0rm** [reply]
You can add some padding to your chart if you want the values to be displayed. I think this should point to the correct css property to control that. https://github.com/fair-acc/chart-fx/issues/632#issuecomment-1765907388
